src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -43,12 +43,14 @@ public LoginPasswordlessViewModel()
 
         public ICommand RejectRequestCommand { get; }
 
-        public string LogInAttemptByLabel => string.Format(AppResources.LogInAttemptByXOnY, LoginRequest?.Email, LoginRequest?.Origin);
+        public string LogInAttemptByLabel => hasEmailAndOrigin ? string.Format(AppResources.LogInAttemptByXOnY, LoginRequest.Email, LoginRequest.Origin) : string.Empty;
```

Comment:
Can you have a `LoginRequest` that doesn't have an `Email` or `Origin` and then those properties are updated?
Because unless that scenario could happen I think that here you could have just 
`public string LogInAttemptByLabel => LoginRequest != null ? string.Format(...) : string.Empty;`
and remove `hasEmailAndOrigin`, btw if you were to keep the property, it needs first letter uppercase `HasEmailAndOrigin`